secant/Util/NavigationLinkModifier.swift

```diff
@@ -0,0 +1,41 @@
+import SwiftUI
+
+struct NavigationLinkModifier<Destination: View>: ViewModifier {
+    @Binding var isActive: Bool
+    var useBodyAsLabel: Bool
```

Comment:
I totally had your suggestion at the beginning! (I called it `emptyNavigationLink`) then I unified them, and to be honest I wasn't sure which I preferred 😅. Seperate feels more "correct", whereas unified was "cleaner" but yeh the default argument is not the nicest. I think I'll try seperate again, if I can get the right naming then it will be the best option 😄

Reply:
separated these, now in 43d3255c178e4a31f171acdd85c4e314566d63ed